### PR TITLE
Verifying the integrity of the options the user wants to save for a section #9

### PR DESCRIPTION
**Pull request by mahfouzhseineweev**
Verifying the integrity of the options the user wants to save for a section #9

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,3 +57,14 @@
...
  - Proper 404 page when the page does not exist eweev/geeks/section-website#67
+
+
+### v1.0.5 (6/7/2023)
+
+---
+
+ - Verifying the integrity of the options the user wants to save for a section #9
+
+ - Global gestion of the errors if a page is saved and a sections is invalid, from the Library #12
```
Comment (jfayad) [inline CHANGELOG.md:68]: gestion is not an english term

File: geeks.solutions-vue-components-1.0.3.tgz
Comment (jfayad) [inline geeks.solutions-vue-components-1.0.3.tgz:1]: this file should not belong to the repo

File: lib/src/components/Sections/index.vue [hunk 2]
```diff
@@ -1690,62 +1695,129 @@ export default {
...
+                  type.fields.forEach(field => {
+                    section.options.forEach(option => {
+                      if (Object.keys(option).includes(field.name)) {
+                        if(option[field.name] && (Array.isArray(option[field.name]) || typeof option[field.name] === 'object')) {
+                          if (Array.isArray(option[field.name])) {
+                            if ((!option[field.name][0].media_id || !option[field.name][0].url) && option[field.name].length !== 0) {
+                              integrityCheck = false
+                              this.loading = false;
+                              this.showToast(
+                                "Error saving your changes",
```
Comment (jfayad) [inline lib/src/components/Sections/index.vue:1714]: the error titles should also be translated

File: package.json [hunk 3]
```diff
@@ -49,7 +49,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@geeks.solutions/vue-components": "^1.0.2",
+    "@geeks.solutions/vue-components": "geeks.solutions-vue-components-1.0.3.tgz",
```
Comment (jfayad) [inline package.json:52]: don't forget to point back to the npm release

**Review by jfayad — CHANGES_REQUESTED** [review]

File: lib/src/components/Sections/index.vue (hunk 2 above)
Comment (mahfouzhseineweev) [reply lib/src/components/Sections/index.vue:1714]: For now, after replacing the toast component for nuxt sections, it does not show any title which is why all titles were kept without translations

**Review by jfayad — APPROVED** [review]

**Pull request merged**